src/wrapper/mod.rs

```diff
@@ -80,6 +83,34 @@ pub fn run() -> i32 {
         }
     };
 
+    let tty_fd = STDIN_FILENO;
+    let is_interactive = isatty(tty_fd).unwrap_or(false);
+    if is_interactive {
```

Comment:
We should always make wrapper a group leader otherwise some tests is hung. The reason is that `vagga _build` that is run after `vagga _run` will be stopped when accessing stdin due to lack of controlling tty.